app/views/state_file/questions/email_address/edit.html.erb

```diff
@@ -6,7 +6,7 @@
 
   <%= form_with model: @form, url: { action: :update }, local: true, method: "put", builder: VitaMinFormBuilder do |f| %>
     <div class="white-group">
-    <%= f.cfa_input_field(:email_address, t(".email_address_label"), help_text: 'example@email.com', classes: ["form-width--long"]) %>
+    <%= f.cfa_input_field(:email_address, t(".email_address_label"), help_text: 'example@email.com', classes: ["form-width--long"], type: "email", options: { autocomplete: "email", required: true }) %>
```

Comment:
does this actually add `aria-required` flag to this element?

Reply:
`aria-required` is no longer necessary because we are applying the required attribute for all users. Sorry for not calling out this implementation choice!

Reply:
huh, but what does the `required: true` do here?